L/LLVMExtra/build_tarballs.jl

```diff
@@ -4,16 +4,16 @@ using Base.BinaryPlatforms
 include("../../fancy_toys.jl")
 
 name = "LLVMExtra"
-repo = "https://github.com/vchuravy/LLVMExtra.git"
-version = v"0.0.1"
+repo = "https://github.com/maleadt/LLVM.git"
```

Comment:
```suggestion
repo = "https://github.com/maleadt/LLVM.jl.git"
```

But what did **you** waste an hour on?